src/main/java/seedu/address/commons/util/XmlUtil.java

```diff
@@ -68,4 +69,27 @@
         m.marshal(data, file.toFile());
     }
 
+    /**
+     * Saves the data in the file in xml format.
```

Comment:
Javadoc can explain the difference between `saveDataToFile` and `saveDataToFileIfMissing`

Reply:
fixed in latest commit